### DISCUSSION TITLE
[Question] Why these emojis are gray

### DISCUSSION THREAD

**Discussion by shink** [community/community#8638] [Other Feature Feedback, Questions, & Ideas]
Big thanks to @github for this awesome feature!

I've a small question, why my emojis are gray?

![1](https://user-images.githubusercontent.com/49934421/145514786-53575aa2-06b3-4044-b019-a4eef6d19e74.png)

![2](https://user-images.githubusercontent.com/49934421/145514721-eed5290a-2ed2-4bcb-a251-3378f2a2a0f3.png)

**Comment by brianlovin** [comment]
Strange! Thanks for the report — I'm assuming you're on Windows, right? What browser are you using?

**Reply by shink** [reply]
@brianlovin Thanks for your reply! Yeah, I'm on Windows 10.

> OS: Windows 10 Professional 21H2
> Browser: Google Chrome 96.0.4664.93 (Stable)

BTW, it's correct on my Ubuntu 20.04 Desktop, That's very strange.

**Reply by loic-sharma** [reply]
I can reproduce this on Windows 10 Edge:

```
Windows 10 version 21H1 (OS Build 19043.1415)
Edge Version 96.0.1054.53 (Official build) (64-bit)
```

**Comment by Katsute** [comment]
Possibly caused by https://bugs.chromium.org/p/chromium/issues/detail?id=1266022
